src/packages/v4/hooks/useNativeTokenSymbol.ts

```diff
@@ -0,0 +1,22 @@
+import { useJBChainId } from 'juice-sdk-react'
+import {
+  arbitrumSepolia,
+  baseSepolia,
+  optimismSepolia,
+  sepolia
+} from 'viem/chains'
+/**
+ *
+ * @todo add to sdk
+ */
+export function useNativeTokenSymbol() {
+  const symbols: { [k: number]: string } = {
+    [sepolia.id]: 'SepETH',
+    [optimismSepolia.id]: 'OPSepETH',
+    [arbitrumSepolia.id]: 'ArbSepETH',
+    [baseSepolia.id]: 'BaseSepETH',
+  }
+
+  const chainId = useJBChainId()
```

Comment:
JBChainId is whatever we pass in to the JBProjectProvider. Its used for all the contract reads (e.g. do all the reads on `sepolia` instead of `mainnet`.

Reply:
Ok got it